network/p2p/p2p_test.go

```diff
@@ -409,38 +342,44 @@ func (r *dummyRouter) Route(_ context.Context, _ *queue.DecodedSSVMessage) {
 	atomic.AddUint64(&r.count, 1)
 }
 
-func dummyMsg(t *testing.T, pkHex string, height int, role spectypes.BeaconRole) (spectypes.MessageID, *spectypes.SignedSSVMessage) {
+func dummyMsg(t *testing.T, pkHex string, height int, role spectypes.RunnerRole) (spectypes.MessageID, *spectypes.SignedSSVMessage) {
 	pk, err := hex.DecodeString(pkHex)
 	require.NoError(t, err)
+
 	id := spectypes.NewMsgID(networkconfig.TestNetwork.Domain, pk, role)
-	signedMsg := &specqbft.SignedMessage{
-		Message: specqbft.Message{
-			MsgType:    specqbft.CommitMsgType,
-			Round:      2,
-			Identifier: id[:],
-			Height:     specqbft.Height(height),
-			Root:       [32]byte{0x1, 0x2, 0x3},
-		},
-		Signature: []byte("sVV0fsvqQlqliKv/ussGIatxpe8LDWhc9uoaM5WpjbiYvvxUr1eCpz0ja7UT1PGNDdmoGi6xbMC1g/ozhAt4uCdpy0Xdfqbv"),
-		Signers:   []spectypes.OperatorID{1, 3, 4},
+	qbftMsg := &specqbft.Message{
+		MsgType:    specqbft.CommitMsgType,
+		Round:      2,
+		Identifier: id[:],
+		Height:     specqbft.Height(height),
+		Root:       [32]byte{0x1, 0x2, 0x3},
 	}
-	data, err := signedMsg.Encode()
+	data, err := qbftMsg.Encode()
 	require.NoError(t, err)
+
 	ssvMsg := &spectypes.SSVMessage{
 		MsgType: spectypes.SSVConsensusMsgType,
 		MsgID:   id,
 		Data:    data,
 	}
-	signedSSVMsg, err := spectypes.SSVMessageToSignedSSVMessage(ssvMsg, 1, dummySignSSVMessage)
-	require.NoError(t, err)
+
+	signedSSVMsg := &spectypes.SignedSSVMessage{
+		Signatures:  [][]byte{[]byte("sVV0fsvqQlqliKv/ussGIatxpe8LDWhc9uoaM5WpjbiYvvxUr1eCpz0ja7UT1PGNDdmoGi6xbMC1g/ozhAt4uCdpy0Xdfqbv")},
+		OperatorIDs: []spectypes.OperatorID{1, 3, 4},
+		SSVMessage:  ssvMsg,
+	}
+
+	// TODO: remove or use?
+	//signedSSVMsg, err := spectypes.SSVMessageToSignedSSVMessage(ssvMsg, 1, dummySignSSVMessage)
+	//require.NoError(t, err)
```

Comment:
why cant we call this instead of producing our own here?

Reply:
Probably because of the signature, but I reverted this for now